### PR TITLE
Allow to specify port for dest

### PR DESCRIPTION
**Pull request by nuclon**
port at real server isn't always the same as service port

### CODE REVIEW

File: tests/ipvs_tests.py [hunk 1]
```diff
@@ -211,6 +211,20 @@ def test_dest_port(self):
                 'Service port {1}.'.format(dest.port(), service.port())
             )
 
+    def test_dest_diff_port(self):
+        self.client.add_service('1.1.1.1', 80)
+        self.client.add_dest('1.1.1.1', 80, '2.2.2.1', 8080)
+        self.client.add_dest('1.1.1.1', 80, '2.2.2.2', 8080, weight=100)
```
Comment (chantra) [inline tests/ipvs_tests.py]: could you add a test where we have 2 destinations with the same ip and different ports.

**Comment by chantra** [comment]
Thanks @nuclon for getting this diff up.

LGTM. There could potentially be multiple destination from the having the same ip but running on different ports so it would be nice if you could add a test for this.

Once the diff is good to go, it would be nice if you could squash the commits in one before we merge the PR.

**Comment by nuclon** [comment]
Done.

**Comment by chantra** [comment]
Thanks @nuclon !

**Pull request merged**